.github/releases/wasm-v0.5.0.md

```diff
@@ -0,0 +1,3 @@
+This release introduces multiple breaking changes to the structure of IOTA DID Documents and their Tangle messages, rendering any identity created with a prior version incompatible and unresolvable. A versioning system has been introduced so any new identities should hopefully be forward compatible with any future breaking changes to the message structure.
+
+The main feature of this release is the introduction of WebAssembly (Wasm) bindings for the high-level `Account` API for Javascript/Typescript in both Node.js and the browser. This includes Stronghold storage support but only for Node.js, as it was determined that compiling Stronghold to Wasm for private key storage in the browser would not be sufficiently secure.
```

Comment:
Should match the Rust release notes exactly; diff chain deprecation and `MerkleKeyCollection` removal affects all bindings too. 

The `Account` Wasm bindings should be mentioned in both.